src/parser/shared/modules/StatTracker.js

```diff
@@ -291,7 +291,6 @@ class StatTracker extends Analyzer {
     [SPELLS.BLUR_OF_TALONS_BUFF.id]: BLUR_OF_TALON_STATS,
     // endregion
     // region Warlock
-    [SPELLS.EXPLOSIVE_POTENTIAL.id]: { haste: 841 },
```

Comment:
Is this intentional?

Reply:
Yes, it's yet another trait so fixed haste on its own is wrong, and it's added correctly in #2751